### ISSUE TITLE
Final review

### ISSUE THREAD

**Issue by masongallo** [rogeliozunigarubio/data_science_final_project#4]
Clarity of research problem:

I'm not 100% clear on your research goal from your slides. How would someone use your results?

Code:

Your notebook is very clear. I'm a bit concerned about the quality of your data. The warning from gensim is sometimes indicative of a dataset that can't be "well separated". Looking at the topics, we see a ton of overlap. 

Next steps:

I think the next step here is evaluating how "good" your models are. One way to do this is to do a test/train split and create a custom distance function. Fit and tune the LDA (or other models) on the train data and score after applying to the test data. For example, your custom distance function could score how separated each topic is.

**Comment by rogeliozunigarubio** [comment]
thank you, I am concerned about the data as well, I want to spend time in cleaning up the data more and have a few engineering ideas that I think will help, which I am working on right now, although they'll take time.

In terms of the clarity of the problem, I think I dumbed it down way too much as a result of discussions with my teammates, their feedback was great but I didn't find a good way to be concise and explain correctly at the same time, I want to include more concrete background in the presentation, a section called motivation, because my feeling is that even if the data available is not good enough I want to lay the groundwork for a useful strategy and present that as my path forward after class is over, which leaves me in the difficult position of not  having very conclusive results, but a clear way to improve once I have better data.

does that make sense?

anywhere I can look for hints on " score how separated each topic is"?

I am also adding an actual review sentiment score using Vader, since it was part of the original idea, which is why I decided to take the extra time to engineer the data capture more so I can have a bigger/better set.

thanks again!
